.github/workflows/main.yml

```diff
@@ -138,7 +138,7 @@ jobs:
       - run: cargo fmt --all --check --manifest-path bonsai/Cargo.toml
       - run: cargo fmt --all --check --manifest-path examples/Cargo.toml
       - run: cargo fmt --all --check --manifest-path tools/crates-validator/Cargo.toml
-      - run: cargo clippy -p risc0-zkvm
+      - run: cargo clippy --workspace --exclude doc-test --exclude fibonacci-methods --exclude risc0-zkvm-methods --exclude xtask
```

Comment:
Why do we need exclusions?

Reply:
For the listed packages above I'm getting compilation error like the following:

```
risc0-zkvm-methods-guest:    Compiling risc0-zkvm-methods-guest v0.1.0 (/home/matheus/git/github/risc0/risc0/risc0/zkvm/methods/guest)                                                                            
risc0-zkvm-methods-guest: error[E0463]: can't find crate for `core`
risc0-zkvm-methods-guest:   |
risc0-zkvm-methods-guest:   = note: the `riscv32im-risc0-zkvm-elf` target may not be installed
risc0-zkvm-methods-guest:   = help: consider downloading the target with `rustup target add riscv32im-risc0-zkvm-elf`
risc0-zkvm-methods-guest:   = help: consider building the standard library from source with `cargo build -Zbuild-std`
risc0-zkvm-methods-guest: 
risc0-zkvm-methods-guest: For more information about this error, try `rustc --explain E0463`.
risc0-zkvm-methods-guest: error: could not compile `risc0-zkvm-methods-guest` (bin "slice_io") due to previous error
risc0-zkvm-methods-guest: warning: build failed, waiting for other jobs to finish...
risc0-zkvm-methods-guest: error: could not compile `risc0-zkvm-methods-guest` (bin "hello_commit") due to previous error
risc0-zkvm-methods-guest: error: could not compile `risc0-zkvm-methods-guest` (bin "multi_test") due to previous error
error: failed to run custom build command for `risc0-zkvm-methods v0.22.0-alpha.1 (/home/matheus/git/github/risc0/risc0/risc0/zkvm/methods)`
```

Maybe there's something I should do on my local dev environment in order to provide `riscv32im-risc0-zkvm-elf` support? 

I'd imagine I already have it though (?)

Reply:
I'm not sure why `clippy` fails though as if I run `cargo check` everything compiles correctly.